README.md

```diff
@@ -5,15 +5,17 @@ Triangle DevOps organizers.
 
 ## Organizers
 
-* Mark Mzyk
-  * GitHub: mmzyk
-  * mmzyk (at) programmersparadox (dot) com
+* Chris Short
+  * GitHub: [chris-short](https://github.com/chris-short)
+  * Twitter: [@ChrisShort](https://twitter.com/ChrisShort)
+  * chris (at) chrisshort (dot) net
+* Chris Bailey
```

Comment:
If Chris Bailey is on the hot list to be contacted, can we get contact information here for him?